packages/paste-website/src/components/homepage/Experiment.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {styled, themeGet} from '@twilio-paste/styling-library';
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
+
+const StyledBrandGradient = styled(Box)`
```

Comment:
Nice, a styled box is a great interim solution until we figure out how to tackle gradients. And still using the theme values as the steps 👍